fuzz/fuzz_targets/client_initial.rs

```diff
@@ -17,18 +17,23 @@ fuzz_target!(|data: &[u8]| {
     };
 
     let mut client = default_client();
-    let ci = client.process(None, now()).dgram().expect("a datagram");
+    let ci = client
+        .process_alloc(None, now())
+        .dgram()
+        .expect("a datagram");
```

Comment:
Note `process_alloc` is simply a convenience function for tests, allocating into an ad-hoc `write_buffer`.